src/enzo/Makefile

```diff
@@ -143,8 +141,8 @@ MACHNOTES:
 	@(if [ -e $@ ] ; then  \
 		echo "Success!" ; \
 		echo "Compiled enzo from " ; \
-		echo "Mercurial Branch   `$(SVN) identify -b`" ; \
-		echo "Mercurial Revision `$(SVN) identify -i`" ; \
+		echo "Mercurial Branch   `git rev-parse --abbrev-ref HEAD`" ; \
```

Comment:
Maybe change: "Mercurial" -> "Git" here?

Reply:
Good catch! I've fixed this in PR #55.